docs/guides/connecting_to_your_data/datasource_configuration/components/intro_for_single_or_multi_batch_configuration.mdx

```diff
@@ -0,0 +1,7 @@
+This means that when configuring your Data Connector's regular expression, you have the option to design it so that the Data Connector is only capable of returning a single Batch per Data Asset, or so that it is capable of returning multiple Batches grouped into individual Data Assets.  Each type of configuration is useful in certain cases, so we will provide examples of both.
```

Comment:
Minor comment: some thing about 'design' and RegEx doesn't quite fit :D maybe build or implement? Again not a big deal.

Reply:
I like "implement" better, too.  I'll change it to that.